eos/b-decays/b-to-3l-nu_TEST.cc

```diff
@@ -0,0 +1,82 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Danny van Dyk
+ * Copyright (c) 2022 Stephan Kürten
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <test/test.hh>
+#include <eos/b-decays/b-to-3l-nu.hh>
+
+using namespace test;
+using namespace eos;
+
+class BToThreeLeptonsNeutrinoTest :
+    public TestCase
+{
+    public:
+        BToThreeLeptonsNeutrinoTest() :
+            TestCase("b_to_3l_nu_test")
+        {
+        }
+
+        virtual void run() const
+        {
+
+            Parameters p = Parameters::Defaults();
+
+            {
+
+                BToThreeLeptonsNeutrino three_lnu(p, Options{ });
+
+                TEST_CHECK_NEARLY_EQUAL(3.20015165332841e-21, three_lnu.double_differential_decay_width(0.6, 1.2), 1e-26);
```

Comment:
Against what implementation is this tested. Please add the information in a comment.